lib/routes/data/styles.js

```diff
@@ -63,4 +63,21 @@ module.exports = {
       ],
     ],
   },
+  breadcrumb: {
+    item: [
+      {
+        title: 'Dashboard',
```

Comment:
Not sure if it's for this PR, but we would need a way to get these titles. Perhaps set up navigation in `config`? Not sure...needs discussion.